template/[% if docs_type == 'sphinx' %]docs[% endif %]/[% if repo_name == 'ss-cpp' %]topics[% endif %]/overlays.md

```diff
@@ -26,3 +26,13 @@ References:
 
 - <https://learn.microsoft.com/en-us/vcpkg/concepts/overlay-ports>
 - [Why overlay ports? Install locally modified package](https://learn.microsoft.com/en-us/vcpkg/consume/install-locally-modified-package)
+
+## Overlay of a custom chinaload toolchain
```

Comment:
same typo here, may need to check globally.

Reply:
I have checked codebase globally, and it only has these two.